File: comms/gfax/files/patch-src_gui.cs
--- src/gui.cs.orig	Mon Feb 28 16:08:34 2005
+++ src/gui.cs	Sun Oct 23 17:31:17 2005
@@ -541,8 +541,8 @@
 		private int update_queue_status(string queue)
 		{
 			G_ListView view;
-			Fax.FaxQueue q;
-			Fax.FaxRecQueue rq;
+			Fax.FaxQueue q = null;
+			Fax.FaxRecQueue rq = null;
 						
 			view = lv;
 				

File: games/macopix/files/patch-src-gui.c
--- src/gui.c.orig	2008-06-24 06:44:20.000000000 +0200
+++ src/gui.c	2013-09-13 14:36:40.096276204 +0200
@@ -4557,7 +4557,7 @@
   gchar *dest_file;
 
   if(flagChildDialog){
-    return;
+    return(NULL);
   }
   else{
     flagChildDialog=TRUE;
@@ -4637,7 +4637,7 @@
   char win_title[64];
 
   if(flagChildDialog){
-    return;
+    return(NULL);
   }
   else{
     flagChildDialog=TRUE;
